demo/demo.py

```diff
@@ -105,8 +104,6 @@ def init(loop):
     app.router.add_route('GET', '/call_ajax', call_ajax, name='call_ajax')
 
     # templates error handlers
-    app.router.add_route('GET', '/mako_exc', test_mako_exc,
```

Comment:
If you really want to allow demo run without `aiohttp_mako` installed please do conditional import.
I mean 
```
try:
    import aiohttp_mako
except ImportError:
    aiohttp_mako = None

if aiohttp_mako:
   app.router.add_route('GET', '/mako_exc', test_mako_exc, name='test_mako_exc')
```

index page for demo app should use pattern like: 
```
{% if aiohttp_mako %}
        <p>
          <a href="${app.router['test_mako_exc'].url()}">Mako Exception example</a>
        </p>
{% else %}
        <p>
          aiohttp_mako is not available, please install it by <code><pre>$ pip install aiohttp_mako</pre></code> to see how mako excetion handling works.
        </p>
{% endif %}
```